rented-properties/src/main/java/org/egov/cpt/service/EnrichmentService.java

```diff
@@ -454,16 +454,25 @@ public void enrichpropertyImageCreateRequest(PropertyImagesRequest propertyImage
 
 		if (!CollectionUtils.isEmpty(propertyImagesRequest.getPropertyImagesApplications())) {
 			propertyImagesRequest.getPropertyImagesApplications().forEach(application -> {
-				String gen_application_id = UUID.randomUUID().toString();
-				application.setId(gen_application_id);
-				application.getProperty()
-						.setId(propertyImagesRequest.getPropertyImagesApplications().get(0).getProperty().getId());
-				application.setAuditDetails(propertyAuditDetails);
-
-				application.setCapturedBy(requestInfo.getUserInfo().getName());
-
-				enrichDocuments(application.getApplicationDocuments(), requestInfo, application.getProperty().getId(),
-						application.getId(), application.getTenantId());
+				if (!CollectionUtils.isEmpty(application.getApplicationDocuments())) {
+					if (application.getApplicationDocuments().size() <= 6) {
```

Comment:
- Can we do this in `ValidationService` instead of `EnrichmentService`?
- The value 6 should not be hardcoded here. It should come from mdms value configured [here](https://github.com/Transerve-PwC/mdms-data/blob/master/data/ch/rented-properties/documents.json#L21).
- There is both upper and lower limit validation.
- Verify that changing the value in mdms changes the validation correctly.